lib/fizzbuzz.rb

```diff
@@ -1,9 +1,15 @@
 class SuperFizzBuzz
 
   def run(input)
-
-    #Implement your code here
-
+    if (input % 3 == 0) && (input % 5 == 0) 
+      "FizzBuzz"
+  	elsif input % 5 == 0
+  	  "Buzz"
+    elsif input % 3 == 0
+  	  "Fizz"
+  	else
+  	  input
+  	end 
   end
```

Comment:
Please make your indentation for each if, elsif, and else consistent. Initially this made me think the first elsif would only hit if the first if evaluated to true for both checks.

Also, you don't need parenthesis around separate compare statements. Still works with them there, but I believe the convention is to not use them, especially for such small and simple comparisons.